database/src/main/resources/db/migration/V1.94.0__RHCLOUD-29443_remove_secrets_columns.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE
+    "camel_properties"
+DROP COLUMN
+    "basic_authentication",
+DROP COLUMN
+    "secret_token";
+
+ALTER TABLE
+    "endpoint_webhooks"
+DROP COLUMN
+    "basic_authentication",
+DROP COLUMN
+    "secret_token";
```

Comment:
We can also drop the "bearer token" column.

Reply:
Or maybe not, `WebhookProperties#bearerAuthentication` is still on `master`.

Reply:
Or maybe not, `WebhookProperties#bearerAuthentication` is not marked as `@Transient` on master.

Reply:
Sounds good 👍